### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by maelle**
Related to #75

Once the testing "strategy" is set I can add tests "with Bootswatch".

### CODE REVIEW

**Comment by maelle** [comment]
I had an issue with https://github.com/r-lib/pkgdown/issues/2019

File: tests/testthat/test-mathjax.R [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    rstudio = FALSE,
+    open = FALSE
+  )
+  usethis::local_project(proj)
+  usethis::use_readme_md()
+  usethis::use_pkgdown()
+  yaml <- yaml::read_yaml("_pkgdown.yml")
+  yaml$mathjax <- TRUE
+  yaml::write_yaml(yaml, "_pkgdown.yml")
+  expect_output({
```
Comment (maelle) [inline tests/testthat/test-mathjax.R:15]: this is just to avoid the messages printed by pkgdown. (no way to toggle them for now if I follow correctly)

**Comment by jeroen** [comment]
Thanks! This looks like a good start to add some tests as we fix common issues

**Comment by maelle** [comment]
well a Bootswatch one at least :wink:

**Pull request merged**